src/language/ast.js

```diff
@@ -44,6 +44,30 @@ export type Location = {
 };
 
 /**
+ * Represents the different kinds of tokens in a GraphQL document.
+ */
+export type TokenKind = '<SOF>'
```

Comment:
I like that better. I moved the `Token` type as it is on master into the `lexer.js` file.